### PR TITLE
Test on Pyhon 3.7 and 3.8 as well (next to 2.7, 3.5, 3.6).

### PR DESCRIPTION
**Pull request by mauritsvanrees**

### CODE REVIEW

**Review by mgedmin — APPROVED** [review]
LGTM!

File: CHANGES.rst [hunk 1]
```diff
@@ -14,7 +14,7 @@ New features:
 
 Bug fixes:
 
-- *add item here*
+- Test on Pyhon 3.7 and 3.8 as well (next to 2.7, 3.5, 3.6).  [maurits]
```
Comment (jamadden) [inline CHANGES.rst]: There's a small typo here. Also, links to issues/PRs are very appreciated.
```suggestion
- Test on Python 3.7 and 3.8 as well (next to 2.7, 3.5, 3.6).  See `PR #19 <https://github.com/zopefoundation/z3c.relationfield/pull/19/>`_.
```

**Review by jamadden — APPROVED** [review]
There's a small typo in the change note.

**Comment by mauritsvanrees** [comment]
Thanks for the reviews. I have committed Jason's suggestion.

**Pull request merged**